airflow/www/templates/airflow/graph.html

```diff
@@ -205,11 +212,11 @@
       container: "body",
     });
 
-    d3.selectAll("div.legend_item.state")
+    d3.selectAll(".legend--state .legend__item")
```

Comment:
Can we be consistent about `--` vs `__` please?

Reply:
I guess 

```suggestion
    d3.selectAll(".legend__state .legend__item")
```

as then `--` is separating the "type" from the value.

Another option is to use an additional attribute instead:

```
     <div class="legend__item" data-legend-item="success">success</div>
```

and then CSS of 
```
.legend__item[data-legend-item="#{$name}"] {
}
```

(Check my CSS syntax there. I did this one from memory)

Reply:
Oh, like you are doing in tree.html :) Lets do that here too.

Reply:
I will update a PR. I had concerns about the specificity of selector Now I checked and it turns out that classes and attributes are counted in the same way..

Reply:
Now we now have the following classes:
```
.legend--pills
.legend__item
.legend__circle
.legend__square
.legend__item--active
.legend--state
```